components/header/avatar.tsx

```
@@ -0,0 +1,46 @@
+          height={dimensions}
+          className="object-cover rounded-full"
+          onError={() => setImageError(true)}
+          priority
+        />
+      </div>
+    );
+  }
+
+  return (
+    <div
+      className={"rounded-full bg-gradient-to-br from-indigo-600 to-indigo-700 flex items-center justify-center text-white font-medium" + sizeMap[size] + className}
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)